### PR TITLE
Update to stage-1

### PR DESCRIPTION
**Pull request by SaraVieira**
Since stage 0 is very unpredictable and quickly changes I think it would be better to update this to stage-1.

All we need is already in stage-1 and it's more predictable

What do you guys think ?

### CODE REVIEW

**Comment by reznord** [comment]
@SaraVieira Yarn support for `preact-cli` is still WIP. You can check the status for that in [here](https://github.com/developit/preact-cli/pull/31).

As far as this yarn.lock is concerned, please don't commit the file, some might use `npm` and some might use `yarn`.

**Comment by prateekbh** [comment]
Will adding it to gitignore not help?

**Comment by reznord** [comment]
That should do !!

File: package.json [hunk 1]
```diff
@@ -84,6 +84,7 @@
     "babel-cli": "^6.24.0",
     "babel-eslint": "^7.2.1",
     "babel-plugin-transform-runtime": "^6.23.0",
+    "babel-preset-stage-1": "^6.24.1",
```
Comment (thangngoc89) [inline package.json]: Could you move this to dependency like stage 0? We need it there, not devDep

**Comment by SaraVieira** [comment]
@lukeed yarn.lock removed and added to the gitignore
@thangngoc89 Also changed

**Comment by developit** [comment]
ooh, the stages changed since I last looked - definitely agreed we don't need Do expressions or the function bind syntax.  good call @SaraVieira!

**Comment by lukeed** [comment]
Anyone know why Node 8 is failing with `cannot find module 'jest-cli'`? I've restarted it once & looks like someone else just did too 😅

**Comment by thangngoc89** [comment]
@lukeed I just restarted it. It failed. When we have a test problem, we shall ping @rkostrzewski

**Comment by SaraVieira** [comment]
Thanks @developit! 

I was gonna ask someone to restart it but apparently it didn't fix it 😐

**Comment by SaraVieira** [comment]
It did pass now 😄 

Is this okay to merge guys ?

**Comment by rkostrzewski** [comment]
Removed cache for build. Commiting `yarn.lock` seems to have changed node_modules structure and cached entities had incorrect structure. Weird.

**Comment by hzoo** [comment]
just as fyi, stage 3 is what is "predictable". Stage 0 is basically automatically ok if someone champions it at TC39, and stage 1 isn't that much different. It's best to opt-in to individual plugins/proposals rather than use stages before Stage 3 but currently we leave it there as an option.

**Pull request merged**